src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -200,7 +246,11 @@ def get_path_to_port_config_file(asic=None):
     if not platform:
         return None
 
-    (platform_path, hwsku_path) = get_paths_to_platform_and_hwsku_dirs()
+    if hwsku:
+        platform_path = get_path_to_platform_dir()
+        hwsku_path = hwsku
+    else:
+        (platform_path, hwsku_path) = get_paths_to_platform_and_hwsku_dirs()
```

Comment:
`hwsku_path = hwsku` is not correct. Suggest changing logic to:

```python
platform_path = get_path_to_platform_dir()
hwsku_path = os.path.join(platform_path, hwsku) if hwsku else get_path_to_hwsku_dir()
```

Reply:
Thought about it, and changed this to 
if hwsku:
        platform_path = get_path_to_platform_dir()
        hwsku_path = os.path.join(platform_path, hwsku)
    else:
        (platform_path, hwsku_path) = get_paths_to_platform_and_hwsku_dirs()
this way the call to platform_path only gets called once

Reply:
I see. Yeah, the combined function is a bit more efficient.